x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs.tsx

```diff
@@ -30,6 +39,73 @@ export interface AlertRawData {
   _id: string;
 }
 
+const FieldLabel = styled.span`
+  font-weight: ${euiLightVars.euiFontWeightMedium};
+  color: ${euiLightVars.euiTitleColor};
+`;
+
+const ContextsTable: React.FC<{ riskScore?: UserRiskScore | HostRiskScore; loading: boolean }> = ({
```

Comment:
nit: Move this component to a new file.

Reply:
Done in commit [move contexts table to its own file](https://github.com/elastic/kibana/pull/174638/commits/80a005d05062e667c5097b5f7c030acd0cdf523e)